lectures/heavy_tails.md

```diff
@@ -1020,9 +1072,9 @@ Since $r \geq \alpha$, we have $\mathbb E X^r = \infty$.
 ```{exercise}
 :label: ht_ex3
 
-Repeat exercise 1, but replace the three distributions (two normal, one
-Cauchy) with three Pareto distributions using different choices of
-$\alpha$.
+Refer to the [figure](#simulation-figure) in the section titled "Simulations". 
```

Comment:
https://jupyterbook.org/en/stable/content/references.html#reference-section-labels

Reply:
currently, I'm using jupyter lab, it seems that my local machine can't render it properly, most of the case I need to look at the CI/CD to find out whether the syntax its applied or not .....